### ISSUE TITLE
Graphical issues and proposals

### ISSUE THREAD

**Issue by IhateTrains** [loup99/WtWSMS#257]
Please submit all graphical proposals/lacking/oddly displayed graphics in this issue.

**Comment by IhateTrains** [comment]
I will start. New image for Frankish Steel bookmark:
![pick_era_frankish_steel](https://user-images.githubusercontent.com/29546927/27994571-f446d964-64c0-11e7-8076-f6a2bc603867.jpg)

**Comment by loup99** [comment]
> I will start. New image for Frankish Steel bookmark since the current one is a placeholder:
![image](https://user-images.githubusercontent.com/7620180/27994631-020393d4-64c2-11e7-9657-b3dc63cca822.png)

Do note that the displayed bookmarks are also placeholders and may change. However, when it comes to this, I'm not hostile to using that picture, although it may fit more with the "cut" effect.

**Comment by IhateTrains** [comment]
OK, I submiteed the cutout version in my current PR.

**Comment by IhateTrains** [comment]
Proposal: unique picture for Scholae Palatinae creation event
![scholae_palatinae_proposal](https://user-images.githubusercontent.com/29546927/28076747-e6183a54-665f-11e7-8c79-5184ca3d73ec.jpg)

**Comment by loup99** [comment]
I'm not convinced by the need for it, but from where did you get it?

**Comment by IhateTrains** [comment]
I think it's a modded Total War unit. I've also found an image of Byzantine emperor reviewing his troops:
![f10b871aae173bc3dc19ed103faead79](https://user-images.githubusercontent.com/29546927/28078005-b59a8be4-6663-11e7-85b9-a5eaf4a8af22.jpg)

**Comment by loup99** [comment]
We can't use Total War elements. As for the picture, those are Varangians.

**Comment by IhateTrains** [comment]
I know, but it's still more Roman-like than random battle scene.

**Comment by IhateTrains** [comment]
We could use this picture for varangians, too.

**Comment by loup99** [comment]
Not really in my view, neither of those would be portrayed like this.

**Comment by TurtleShroom** [comment]
I agree with Ihatetrains. The first picture would look good in the Mod. The second one is of the Varangarians, though, as indicated by the symbols on their shields.

**Comment by loup99** [comment]
We can't use the first picture for copyright and modding rules reasons. No Total War-derived content. Only the base assets of that modder without any Total war elements with permission.

Concerning the second, the whole imagery is about Vikings.

**Comment by TurtleShroom** [comment]
Okay. That makes sense.

**Comment by loup99** [comment]
In my opinion these legion pictures could serve:
¨
![image](https://user-images.githubusercontent.com/7620180/28130642-b33c39a4-6737-11e7-983e-56080f539a5e.png)

![image](https://user-images.githubusercontent.com/7620180/28130658-bc7d5e44-6737-11e7-9719-7e4161c9a707.png)

![image](https://user-images.githubusercontent.com/7620180/28130669-c3adc6ae-6737-11e7-8b1e-338a15feccc0.png)

Although note that these are reenactments and that we would need to use effects such as cut-out, as well as doing image editing.

**Comment by TurtleShroom** [comment]
Are they _accurate_ reenactments, though? Most pictures of Legionaires and Roman soldiers that I see are based on the _first_ century (mohawk helmet, rectangle shield, more bared skin, etc.), which is universally associated with Rome.

**Comment by loup99** [comment]
Those are just example samples. I believe the latest one on Commons was one on the 4th century or so.

**Comment by TurtleShroom** [comment]
Okay.

**Comment by IhateTrains** [comment]
We could also use the existing "legion marching" picture:
![roman_legion_marching](https://user-images.githubusercontent.com/29546927/28131707-0bb7db6c-673b-11e7-8512-c5466530d850.jpg)

**Comment by loup99** [comment]
Good idea! Changing the CoA on that, if possible, is the best solution in my opinion.

**Comment by IhateTrains** [comment]
I'm not skilled enough to change the CoA/entire shields and keep the shades :(  I can only do something like this:
![roman_legion_marching_chi_ro](https://user-images.githubusercontent.com/29546927/28134062-1bf64f48-6742-11e7-88f1-3fdec76ebe9a.jpg)
or this:
![roman_legion_marching_chi_ro_2](https://user-images.githubusercontent.com/29546927/28134071-2453f4ce-6742-11e7-9dd1-e6dccc814d57.jpg)

(maybe intense would hide the lack of shades). Are these ones better than original?

**Comment by loup99** [comment]
I'm on the fence, because it does create a contrast. The shields are also too large on the top one. Otherwise we can use the original, for a lack of a better option.

**Comment by IhateTrains** [comment]
Proposal: unique picture for Little Augustus bookmark:
![pick_era_little_augustus](https://user-images.githubusercontent.com/29546927/28474199-e701f56e-6e47-11e7-8103-ed2a9e7f4d44.jpg)

**Comment by IhateTrains** [comment]
Proposal: revised flag of the Eastern Roman Empire
Before:
![2017-07-21](https://user-images.githubusercontent.com/29546927/28489668-110e1286-6ec8-11e7-928a-63bcacb02cfc.png)
After:
![2017-07-22 1](https://user-images.githubusercontent.com/29546927/28489670-23c2bf94-6ec8-11e7-9778-6efc5685d428.png)

I made Chi-Rho a bit smaller and red a bit darker.

**Comment by loup99** [comment]
Yeah, that is better!

**Comment by IhateTrains** [comment]
Needed: More fitting event picture for 2000th anniversary of the Gothic kingdom

**Comment by IhateTrains** [comment]
Some proposals from the existing files:
royal_court.tga from the mod
feast_attila.dds from the mod
dancing_feast_wol.dds from vanilla
Banquet_hall_Feast.dds from vanilla

**Comment by loup99** [comment]
Having looked at them, I think feast_attila is quite a good fit. Do you have any preferences?

**Comment by IhateTrains** [comment]
It also appeals to me. The vanilla feast pictures are good but are too commonly used.

**Comment by loup99** [comment]
The Legendary Leader trait should be distinct from Guard Veteran.

**Comment by IhateTrains** [comment]
![2017-10-13 2](https://user-images.githubusercontent.com/29546927/31564145-13caedc4-b062-11e7-87e6-ab7bdd064bc9.png)

They are slightly different but yeah, not much :D

**Comment by IhateTrains** [comment]
Noted in my TODOs, but I still don't have any good idea on it

**Comment by IhateTrains** [comment]
Proposal: Chi-Rho flag for the Western Roman Empire
I suggest giving the current red Chi-Rho to e_wre and making a purple version for e_byzantium.
Do you want a preview?

**Comment by loup99** [comment]
I disagree. The Chi Roh was formed by Greek letters, and the current flag is a lot more styled. It also allows to distinguish the two.

**Comment by IhateTrains** [comment]
Needed: icon for Samaritan religion, as it is no longer just a Jewish heresy

**Comment by loup99** [comment]
I realise that it should be possible to add an unique texture for the Bureaucratic government.

**Comment by loup99** [comment]
Going to split off the "missing" religion icons into a separate file.

**Comment by IhateTrains** [comment]
For the Bureaucratic government we could use the Imperial government icon from vanilla.

**Comment by loup99** [comment]
I don't see how that would be an improvement. I prefer the current icon.

**Comment by IhateTrains** [comment]
Sorry, I thought you were looking for the government icon replacement.

**Comment by loup99** [comment]
Oh, yes, I realise it wasn't obvious from my post, sorry. What I meant are the special government ribbons and map icons for the government, not the government icon which we already have.

**Comment by IhateTrains** [comment]
What would be the color of the ribbons?

**Comment by loup99** [comment]
That could be one alternative, yes. Preferably not the same as any other government form to differentiate though.

**Comment by IhateTrains** [comment]
Proposal: new era image for the Little Augustus era
![image](https://user-images.githubusercontent.com/29546927/41499650-b5ed0776-7183-11e8-8f54-f4ee880752c1.png)
(the dude is Odoacer)

Advantages over the current one:
- related to the era
- fixes the problem with the image being wrongly displayed after selecting the era, introduced in one of the recent patches

**Comment by loup99** [comment]
What is the issue? That can be fixed separately no matter the picture used.

With regards to the proposal, since it clashes with the other ones in terms of graphics and has no real identification for the players (since we don't know exactly how Odoacer looked, although this has the merit of being the closest we can get) I'm personally sceptic. The current one is also "related" to the era in the sense that the player can identify, although naturally it could be improved.

**Comment by dantaki** [comment]
I am running 2.8.0 and there are missing portraits and I cannot click on the person to interact (declare war etc) 
![example](https://user-images.githubusercontent.com/6598195/44304946-7a3e8a00-a320-11e8-937d-d6105ff3b3f6.PNG)

**Comment by loup99** [comment]
@dantaki This is not a place for bug reports, especially for old versions of the game. Please register your game and post in [our sub-forum](https://forum.paradoxplaza.com/forum/index.php?forums/when-the-world-stopped-making-sense-mod.876/).

**Comment by Darokrithia** [comment]
@dantaki Thank you for the post btw.  Also please include the save file when you post it on the sub-forum
